prow/cmd/pipeline/controller.go

```diff
@@ -569,10 +570,10 @@ func makePipelineGitResource(name string, refs prowjobv1.Refs, pj prowjobv1.Prow
 		revision = refs.BaseRef
 	}
 
-	pr := pipelinev1beta1.PipelineResourceType{
+	pr := pipelinev1alpha1.PipelineResource{
```

Comment:
Going to leave all the `PipelineResource` references as is since it is deprecated and the tekton `v1beta1` pkgs refers it in the `v1alpha1` pkg anyway

Reply:
On second thoughts, this needs to be resolved else, the PROW_IMPLICIT_GIT_REF won't work for tekton pipelines